modules/ui/maproulette_details.js

```diff
@@ -0,0 +1,154 @@
+export function uiMapRouletteDetails(context) {
+  const l10n = context.systems.l10n;
+  const map = context.systems.map;
+  const maproulette = context.services.maproulette;
+
+  let _maprouletteTask;
+
+
+  function taskString(d) {
+    if (!maproulette || !d) return '';
+  }
+
+
+  function maprouletteDetails(selection) {
+    const details = selection.selectAll('.error-details')
+      .data(_maprouletteTask ? [_maprouletteTask] : [], d => `${d.id}-${d.status || 0}` );
+
+    details.exit()
+      .remove();
+
+    const detailsEnter = details.enter()
+      .append('div')
+      .attr('class', 'error-details qa-details-container');
+
+
+    // Description
+    if (taskString(_maprouletteTask, 'detail')) {
+      const div = detailsEnter
+        .append('div')
+        .attr('class', 'qa-details-subsection');
+
+      div
+        .append('h4')
+        .html(l10n.tHtml('QA.keepRight.detail_description'));
+
+      div
+        .append('p')
+        .attr('class', 'qa-details-description-text')
+        .html(d => taskString(d, 'detail'))
+        .selectAll('a')
+        .attr('rel', 'noopener')
+        .attr('target', '_blank');
+    }
+
+    // Elements (populated later as data is requested)
+    const detailsDiv = detailsEnter
+      .append('div')
+      .attr('class', 'qa-details-subsection');
+
+    // Suggested Fix (mustn't exist for every issue type)
+    if (taskString(_maprouletteTask, 'fix')) {
+      const div = detailsEnter
+        .append('div')
+        .attr('class', 'qa-details-subsection');
+
+      div
+        .append('h4')
+        .html(l10n.tHtml('map_data.layers.maproulette.fix_title'));
```

Comment:
This is probably just a case where you copied some code from elsewhere that was doing it but.. We're trying to avoid using code like `.html(l10n.tHtml(stringID))` because it circumvents string sanitization - #1274 has more on this.   If these strings somehow ended up dangerous html, it gets inserted directly into the DOM and that's bad because it can do things to the user that we don't intend.  

Anyway, we are cleaning these up as we go, and the risk is pretty minimal, so no worries - we just try not to add it to new code.

Aside:  One thing React got right was - they named this feature `dangerouslySetInnerHTML`  so people would think twice before using it!
https://react.dev/reference/react-dom/components/common#dangerously-setting-the-inner-html